nalgebra-sparse/src/coo.rs

```diff
@@ -45,6 +45,43 @@ pub struct CooMatrix<T> {
     values: Vec<T>,
 }
 
+impl<T: na::Scalar> CooMatrix<T> {
+    /// Pushes a dense matrix into the sparse one.
+    ///
+    /// This adds the dense matrix `m` starting at the `r`th row and `c`th column
+    /// to the matrix.
+    ///
+    /// Panics
+    /// ------
+    ///
+    /// Panics if any part of the dense matrix is out of bounds of the sparse matrix
+    /// when inserted at `(r, c)`.
+    #[inline]
+    pub fn push_mat<R: na::Dim, C: na::Dim, S: nalgebra::storage::Storage<T, R, C>>(
```

Comment:
To add something more than just my personal taste here, the reason is that abbreviations tend to hurt readability a little because the more abbreviations you need to keep in the back of your head, the slower you are able to read the code. Of course, sometimes abbreviations can help code readability by reducing character counts, but at least in my opinion `push_matrix` does not meet this criterion!

Reply:
Contrary to what I used here, I strongly prefer unabbreviated identifiers when possible, so I'd happily go with `push_matrix`. Or even `push_dense_matrix`?

Reply:
https://github.com/dimforge/nalgebra/pull/904/commits/fec32b1de2d95faa7509359326ce2ab89981d489

Reply:
I agree that `push_dense_matrix` is even more precise. I think perhaps we can get away with `push_matrix` here though, since the canonical matrix type in `nalgebra` is still `Matrix`, and so sparse formats rather represent the exceptional case. Also, when reading the code, it doesn't *really* matter if the matrix would be dense or sparse, as mathematically they would accomplish the same thing - so the inferred logic behind the code would be the same. I think my vote goes to `push_matrix`.